### DISCUSSION TITLE
An error has occured in the editor

### DISCUSSION THREAD

**Discussion by ALedl** [knowledgefutures/pubpub#2225] [Support]
Hi,

when trying to go to this draft pub (https://0277.pubpub.org/pub/ailrzesn/draft?collectionSlug=critical-library-perspectives), I get the message: 

****
An error has occured in the editor.
We've logged the error and will look into the cause right away.

All previous changes have been successfully saved.

To continue editing, please refresh the page.
****

By refreshing the page, I get the same message again and editing remains blocked.

**Comment by gabestein** [comment]
Hello, and sorry you faced this issue. On very rare occasions, the editor gets into an invalid state that must be restored by hand. This was one of those cases. It looks like something happened with one of the discussions, which I had to un-anchor to solve the problem. But the document should be restored.
